src/styles/reset.less

```diff
@@ -50,6 +136,7 @@ video {
   border: 0;
   margin: 0;
   padding: 0;
+  transition: background 0.3s ease;
```

Comment:
![Screen Shot 2022-02-27 at 12 41 29 AM](https://user-images.githubusercontent.com/33165426/155875174-7ee2538f-2a2d-4bb1-8b5d-11ab7ff819f0.jpg)
the tiny arrow on the dropdown is also impossible to see in dark mode. would look better in white but it's an ant design component so this might not be possible to fix